src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -290,7 +297,7 @@ else if (entity.isInWater())
                 Vec3d Vec3d = this.getPath().getPosition(this.entity);
 
                 if (Vec3d.squareDistanceTo(new Vec3d(entity.posX, Vec3d.yCoord, entity.posZ)) < 0.1
-                        && Math.abs(entity.posY - Vec3d.yCoord) < 0.5)
+                      && Math.abs(entity.posY - Vec3d.yCoord) < 0.5)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)